docs/usage/self-hosted-configuration.md

```diff
@@ -219,6 +219,12 @@ Results which are soft expired are reused in the following manner:
 - The `etag` from the cached results will be reused, and may result in a 304 response, meaning cached results are revalidated
 - If an error occurs when querying the `npmjs` registry, then soft expired results will be reused if they are present
 
+## checkedBranches
+
+This array will allow you to set the names of the branches you want to rebase/create, as if you selected their checkboxes in the Dependency Dashboard issue.
+
+Example: `renovate --checked-branches=renovate/chalk-4.x renovate-reproductions/checked` will rebase the `renovate/chalk-4.x` branch in the `renovate-reproductions/checked` repository.`
```

Comment:
Quote from the config option code:

```ts
    type: 'array',
    subType: 'string',
```

The docs for this new feature should explain:

- the valid syntax of the command
- the correct order of arguments
- how to see that this is an array (I guess the `=` part takes an array?
- how to put multiple commands into one line in an array
- explain limitations like don't mix and match repositories
- explain that the branch name used as argument must exist on the chosen repository

I'm thinking of something like this:

> For `checkedBranches` to work, you must match this syntax and order:
>
> 1. `renovate` base program
> 2. `--checked-branches=` command, which takes a `put-the-branch-name-here` branch argument
> 3. The repository to run the whole command on: `renovate-reproduction/checked`
>
> For example: `renovate --checked-branches=renovate/chalk-4.x renovate-reproductions/checked` will rebase the `renovate/chalk-4.x` branch in the `renovate-reproductions/checked` repository.`
>
> To change multiple branches on one repository, you would to this:
>
> `......`
>
> The branch name you want to change must exist on the repository.
> The repository name must be valid as well.

Reply:
We don't want to repeat our syntax for every command we have, so this is too much. Arrays are always comma separated in CLI and this is not the first one. I don't think this command needs much more, but potentially our generic CLI
docs do